tests/odov3/odo_test.go

```diff
@@ -315,7 +315,7 @@ func waitForHttp(url string, expectedCode int) error {
 func waitForPort(devError chan error) ([]ForwardedPort, error) {
 	args := []string{"describe", "component", "-o", "json"}
 
-	maxTries := 20
+	maxTries := 50
```

Comment:
I'm wondering here, if we also need to increase the timeout too. If we have such cases that need more than 20 retries, does it make sense to increase it to `3h` instead of `2h` (in the `check_odo3.sh)

Reply:
One other thought I had, is maybe we can keep the same amount of retries but increase the delay?

E.g. first retry 10 secs the second will be bigger and so on (keeping always the overall time of retries to the 50*10 secs time so we don't wait for more time)

Reply:
I don't mind increasing the timeout to `3h` instead of `2h`, however, I did notice in testing that when a stack does timeout (Quarkus for example and that's why it is skipped) it will timeout every single time and all the others will run fine and relatively quickly. I think on average it was between 1h and 1.5h for the `odov3` testing consistently.

For your other point of increasing the delay we can try that, right now I believe every time it runs the delay increases by `10 seconds` from the previous so it would take some testing to see what the delay needs to be to avoid any issues.

In my opinion I like the idea of increasing the timeout from `3h` from `2h` more because the `maxTries` being set to `50` is already producing the results we're after. Increasing the timeout as you said also will give us benefit of having a buffer incase the GitHub runner is being slow or something and takes more time.